system/Database/SQLite3/Connection.php

```diff
@@ -273,7 +273,7 @@ protected function _fieldData(string $table): array
             $retVal[$i]->type        = $query[$i]->type;
             $retVal[$i]->max_length  = null;
             $retVal[$i]->default     = $query[$i]->dflt_value;
-            $retVal[$i]->primary_key = isset($query[$i]->pk) && (bool) $query[$i]->pk;
+            $retVal[$i]->primary_key = $query[$i]->pk ?? false;
```

Comment:
I haven't studied all the code, but why are we setting `false` here? Shouldn't it be `0`?

Reply:
Thanks! I don't know why I wrote `false`. 😅 
Fixed.

Reply:
Thanks!